integration-tests/utils/Applications.ts

```diff
@@ -142,6 +143,7 @@ export class Applications {
 
   static goToIntegrationTestsTab() {
     cy.get(integrationTestsTabPO.clickTab).click();
+    return new IntegrationTestsTabPage();
```

Comment:
🤔 Instead of returning instance here can we import it in basic flow? may be for any other validations we can utilise it if there is a requirement.

Reply:
Sounds good! Updated. (It looks like half of the `goTo*` functions return a `*Page` instance, while the other half does not. I picked the wrong pattern to follow :smile: )